api/src/routes/messenger.ts

```diff
@@ -10,4 +11,8 @@ router.get("/solrindex/:pid", async function(req, res, next) {
     res.send(JSON.stringify(fedoraFields, null, "\t"));
 });
 
+// TODO: best home?
```

Comment:
I don't think this belongs here; the routes are for handling HTTP requests, and I don't think we want to start up a job worker as a side effect of initializing HTTP routes.

Reply:
I was envisioning this as a command-line script that could be started via npm, but I'm open to other ideas... Perhaps even something that could be kept alive using [Supervisor](http://supervisord.org/) or something similar.